pspamm/codegen/architectures/arm/generator.py

```diff
@@ -44,15 +42,18 @@ def get_template(self):
     def make_reg_blocks(self, bm:int, bn:int, bk:int, v_size:int, nnz:int, m:int, n:int, k:int):
         assert(bm % v_size == 0)
         vm = bm//v_size
```

Comment:
Doesn't the assertion above guarantee that `vm` will be an integer? I can add the ceiling division anyway to keep it consistent with the `arm_sve` generator.
The same applies to the `hsw` and `knl` generators.

Reply:
The question is maybe: do we need extra features (like masking) if we want to support the cases where `bm % v_size != 0`?
If it's possible without them, then changing the two lines to a ceiling division could be worth it—otherwise, they're good as they are I'd say.